proto/umee/leverage/v1/tx.proto

```diff
@@ -38,6 +41,12 @@ service Msg {
 
   // SupplyCollateral combines the Supply and Collateralize actions.
   rpc SupplyCollateral(MsgSupplyCollateral) returns (MsgSupplyCollateralResponse);
+
+  // AddTokenToRegistry adds new tokens to the token registry.
+  rpc AddTokensToRegistry(MsgAddTokensToRegistry) returns (MsgAddTokensToRegistryResponse);
+
+  // UpdateRegistry updates existing tokens with new settings.
+  rpc UpdateRegistry(MsgUpdateRegistry) returns (MsgUpdateRegistryResponse);
```

Comment:
Why doing two different messages? The idea I was proposing was to have only one message, but with two lists:
```
repeated Token add = 4 [(gogoproto.nullable) = false];
repeated Token update = 4 [(gogoproto.nullable) = false];
```
It gives us more flexibility: if we want we can create one proposal (and one voting) to update and add some token settings at once, rather than doing 2 proposals. But if needed, we can always use only one of the lists and do 2 proposals.

Reply:
<img width="1155" alt="image" src="https://user-images.githubusercontent.com/17549398/199894547-d9f7f39f-f985-4e3e-a6f1-8fcc623785b7.png">

It is better to have two msg types for update and adding , because in proposal users can understand tokens are updating or adding 

but we can do both in one msg type but it is not much user understandable  (tokens are updating or adding...)

Reply:
> but we can do both in one msg type but it is not much user understandable (tokens are updating or adding...)

Why it's not "much user understandable"? The separation is clear in the proposal is clear. 
In the proposal you can see the full JSON of the message: https://umee.explorers.guru/proposal/26, and you will see which tokens are added which updated, because it will go into 2 different lists.

And of course, any update should bee correctly described. In prop25 we did update the description correctly. So in fact it doesn't matter if it's 2 different msg types or one if the description is wrong. One could make a wrong description (saying that adding tokens A and B) but in fact adding totally different tokens (C and D).

Reply:
```proto
repeated Token add_tokens = 4 [(gogoproto.nullable) = false];
repeated Token update_tokens  = 4 [(gogoproto.nullable) = false];
```

`add_tokens` for new tokens 
`update_tokens` for new settings for existed tokens

@robert-zaremba

Reply:
looks good. (the sequence number is wrong above, but you have it right in the proto file).